LEGO1/lego/legoomni/include/act1state.h

```diff
@@ -25,29 +29,85 @@ class Act1State : public LegoState {
 	MxBool SetFlag() override;                          // vtable+0x18
 	MxResult VTable0x1c(LegoFile* p_legoFile) override; // vtable+0x1c
 
-	inline void SetUnknown18(MxU32 p_unk0x18) { m_unk0x18 = p_unk0x18; }
-	inline MxU32 GetUnknown18() { return m_unk0x18; }
-	inline MxU32 GetUnknown1c() { return m_unk0x1c; }
-	inline MxS16 GetUnknown21() { return m_unk0x21; }
+	inline void SetUnknown18(MxU32 p_unk0x18) { m_unk0x018 = p_unk0x18; }
+	inline MxU32 GetUnknown18() { return m_unk0x018; }
+	inline MxU32 GetUnknown1c() { return m_unk0x01c; }
+	inline MxS16 GetUnknown21() { return m_unk0x021; }
 
-	inline void SetUnknown1c(MxU32 p_unk0x1c) { m_unk0x1c = p_unk0x1c; }
-	inline void SetUnknown21(MxS16 p_unk0x21) { m_unk0x21 = p_unk0x21; }
+	inline void SetUnknown1c(MxU32 p_unk0x1c) { m_unk0x01c = p_unk0x1c; }
+	inline void SetUnknown21(MxS16 p_unk0x21) { m_unk0x021 = p_unk0x21; }
 
 	void FUN_10034d00();
 
 	// SYNTHETIC: LEGO1 0x10033960
 	// Act1State::`scalar deleting destructor'
 
+	// SIZE 0x4c
+	class NamedPlane {
+	public:
+		// FUNCTION: LEGO1 0x10033800
+		NamedPlane() {}
+
+		inline void SetName(const char* p_name) { m_name = p_name; }
+		inline const MxString* GetName() const { return &m_name; }
+		int Serialize(LegoFile* p_file);
```

Comment:
Instead of `int` it should be `MxS32`. there's a few more places

Reply:
Those are goners now.